TCAT/Controllers/RouteDetailViewController.swift

```diff
@@ -413,15 +397,16 @@ class RouteDetailViewController: UIViewController, GMSMapViewDelegate, CLLocatio
 
                 UIView.animate(withDuration: 1.0, delay: 0, options: .curveEaseInOut, animations: {
                     newBus.userData = bus
-                    (newBus.iconView as? BusLocationView)?.setBearing(bus.heading, start: existingBus!.position, end: busCoords)
+                    // (newBus.iconView as? BusLocationView)?.setBearing(bus.heading, start: existingBus!.position, end: busCoords)
```

Comment:
can you delete this?

Reply:
I toggle between `bearing` and `betterBearing`, not sure why you don't have` betterBearing`